Makefile

```diff
@@ -28,7 +28,7 @@ project-carthage:
 	@ruby patch-carthage-project.rb SwiftGRPC-Carthage.xcodeproj || echo "xcodeproj ('sudo gem install xcodeproj') is required in order to generate the Carthage-compatible project!"
 
 test: all
-	swift test $(CFLAGS)
+	swift test
```

Comment:
There are a couple of `nio` related targets in this Makefile, could you delete those as well?

Reply:
Should be done.